modules/admin-top-bar/assets/scss/admin.scss

```diff
@@ -0,0 +1,211 @@
+// Ask Josh
+@import "../../../../assets/dev/scss/helpers/direction";
+@import "../../../../assets/dev/scss/helpers/variables";
+@import "../../../../assets/dev/scss/helpers/mixins";
+@import "../../../../assets/dev/scss/editor/breakpoints";
+
+$top-bar-height: 50px;
+
+// Hide the widget in admin dashboard that was created by checking the screen option checkbox.
+// "display: none" won't work here since it will influence the mechanism for saving the state for the checkbox in screen options
+#e-dashboard-widget-admin-top-bar {
```

Comment:
Just use it instead of your React root. By using it you also don't need to deal with the checkbox changes in `AdminTopBar` component

Reply:
@matipojo To use the widget as the container of admin top bar, I will have to give it a position absolute, but this wont take it to the top of the page since there is a positioned ancestor. giving it a fixed position wont give us the necessary behavior as describe in the task.

Reply:
After a discussion we decided not to do it.